Mathlib/FieldTheory/IsAlgClosed/AlgebraicClosure.lean

```diff
@@ -416,4 +416,25 @@ instance [CharZero k] : CharZero (AlgebraicClosure k) :=
 instance {p : ℕ} [CharP k p] : CharP (AlgebraicClosure k) p :=
   charP_of_injective_algebraMap (RingHom.injective (algebraMap k (AlgebraicClosure k))) p
 
+/-09GY Lemma: Two polynomials in $k[x]$ are relatively prime
```

Comment:
1. Write `/--` instead of `/-`, as this will generate comments on the Mathlib documentation website.
2. There's no need to write “09GY Lemma:”; just mark the Stacks tag on the last line.